Source/WebCore/rendering/RenderLayer.h

```diff
@@ -891,12 +899,14 @@ class RenderLayer : public CanMakeWeakPtr<RenderLayer> {
 
     bool shouldBeNormalFlowOnly() const;
     bool shouldBeCSSStackingContext() const;
+    bool shouldBeBackdropRootCandidate() const;
```

Comment:
"should be candidate" is a bit ambiguous. It's either a candidate or it's not, so I think we need some renaming of these functions. Maybe remove the "candidate" and say `canBeBackdropRoot()`